src/components/portfolio/DetailsVolatilityGraph.tsx

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+import {
+  makeStyles,
+  createStyles,
+  Theme,
+  useTheme,
+} from '@material-ui/core/styles';
+import FlashOnIcon from '@material-ui/icons/FlashOn';
+import HotelIcon from '@material-ui/icons/Hotel';
+import Grid from '@material-ui/core/Grid';
+import VolatilityLineEntry from './VolatilityLineEntry';
+import { Position } from '../../portfolio/APIClient';
+
+type DetailsVolatilityGraphProps = {
+  positions: Position[];
+};
+
+const DetailsVolatilityGraph: React.FC<DetailsVolatilityGraphProps> = ({
+  positions,
+}) => {
+  const useStyles = makeStyles(({ palette }: Theme) =>
```

Comment:
I don't know if it's a good idea to put this inside of the component for performance reasons. 
@JulianBouchard Don't we usually have these outside of the component?

Reply:
Oh yeah I must have misplaced this in the Graph + the Line component. Let me fix this real quick